man/systemd.resource-control.xml

```diff
@@ -736,6 +736,27 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DisableControllers=</varname></term>
+
+        <listitem>
+          <para>Disables controllers from being enabled for a unit's children. If a controller listed is already in use
+          in its subtree, the controller will be removed from the subtree. This can be used to avoid child units being
+          able to implicitly or explicitly enable a controller. Defaults to not disabling any controllers.</para>
+
+          <para>It may not be possible to successfully disable a controller if the unit or any child of the unit in
```

Comment:
Why was this child-over-parents policy chosen? I can see that disabling and delegating a controller on the same unit is non-sensical config, however, I fail to see why children can override with their Delegate= requests.

(Posting to the commit as it's closest to the origin, not sure if this sends notifications, so pasting a handle @cdown. Let me know if this dicussion should be redirected elsewhere.)

Reply:
Oh, I realize. We'd need to know the subtree of delegated children in order to do the "bottom-up" disabling. (Which we in theory know but we've commited not to interfere with Delegate=.)